.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/rust
+{
+  "name": "Rust",
+  "image": "mcr.microsoft.com/devcontainers/rust:1-1-bullseye",
+  "onCreateCommand": ".devcontainer/oncreate",
+  "features": {
+    "ghcr.io/devcontainers/features/azure-cli:1": {},
+    "ghcr.io/devcontainers/features/github-cli:1": {},
+    "ghcr.io/devcontainers/features/sshd:1": {}
```

Comment:
Why? This opens a can of worms I think we should avoid given all the ways you can connect to a devcontainer currently.

Reply:
[It's part of the requirements in order to be able to SSH to a Codespace](https://docs.github.com/en/codespaces/developing-in-a-codespace/using-github-codespaces-with-github-cli#ssh-into-a-codespace). I'm OK removing it if you don't feel that's necessary.

Reply:
Clarifying a bit.

I do often SSH to codespaces, to get a full-screen terminal or to run TUI apps like [lazygit](https://github.com/jesseduffield/lazygit) (which I'll admit I use more than most, as a 🤓 Big Nerd 🤓 ). So I do prefer having it available, but if you're concerned about other aspects of it, I'm OK removing it. 

The SSHD feature works through the existing secure port forwarding supported by Codespaces, and the `gh` CLI has native support for it, so we're not talking about Codespaces with port `22` open to the internet.

Reply:
Thanks for clarifying. I'm fine keeping it in, then. I was afraid it was basically just stock sshd which - surprisingly - still has PasswordAuthentication enabled - at least in Ubuntu - last I checked.